examples/pytorch/mvgrl/README.md

```diff
@@ -0,0 +1,144 @@
+# DGL Implementation of MVGRL
+This DGL example implements the model proposed in the paper [Contrastive Multi-View Representation Learning on Graphs](https://arxiv.org/abs/2006.05582).
+
+Author's code: https://github.com/kavehhassani/mvgrl
+
+## Example Implementor
+
+This example was implemented by [Hengrui Zhang](https://github.com/hengruizhang98) when he was an applied scientist intern at AWS Shanghai AI Lab.
+
+## Dependencies
+
+- Python 3.7
+- PyTorch 1.7.1
+- dgl 0.6.0
+- networkx
+- scipy
+
+## Datasets
+
+##### Unsupervised Graph Classification Datasets:
+
+ 'MUTAG', 'PTC_MR', 'IMDB-BINARY', 'IMDB-MULTI', 'REDDIT-BINARY', 'REDDIT-MULTI-5K' of dgl.data.GINDataset.
+
+| Dataset         | MUTAG | PTC_MR | RDT-B  | IMDB-B | IMDB-M |
+| --------------- | ----- | ------ | ------ | ------ | ------ |
+| # Graphs        | 188   | 344    | 2000   | 1000   | 1500   |
+| # Classes       | 2     | 2      | 2      | 2      | 3      |
+| Avg. Graph Size | 17.93 | 14.29  | 429.63 | 19.77  | 13.00  |
+
+##### Unsupervised Node Classification Datasets:
+
+'Cora', 'Citeseer' and 'Pubmed'
+
+| Dataset  | # Nodes | # Edges | # Classes |
+| -------- | ------- | ------- | --------- |
+| Cora     | 2,708   | 10,556  | 7         |
+| Citeseer | 3,327   | 9,228   | 6         |
+| Pubmed   | 19,717  | 88,651  | 3         |
+
+
+## Arguments
+
+##### 	Graph Classification:
+
+```
+--dataname         str     The graph dataset name.                Default is 'MUTAG'.
+--gpu              int     GPU index.                             Default is -1, using cpu.
+--epochs           int     Number of training periods.            Default is 200.
+--batch_size       int     Size of a training batch.              Default is 64.
```

Comment:
Starting from `batch_size`, the order of arguments is inconsistent with the ones in `graph/main.py`.